qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -71,6 +74,12 @@ class DefaultPoliciesTest extends BaseSpecification {
             "webhookserver - Kubernetes Actions: Port Forward to Pod",
     ]
 
+    static final private String STRUTS_IMG = "quay.io/rhacs-eng/qa:struts-app"
```

Comment:
What are these for? I don't see them being used anywhere.

Reply:
Yeah, intended to use the coasts, just extracted the images to delete from the deployments.